_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -0,0 +1,167 @@
+- [ARIA4: Using a WAI-ARIA role to expose the role of a user interface component](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA4)
+- [ARIA6: Using aria-label to provide labels for objects](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6)
+- [ARIA10: Using aria-labelledby to provide a text alternative for non-text content](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA10)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+The `<svg>` element has an explicit role of `img` and an accessible name from the `<title>` elememt that does not contain only whitespace.
+
+```html
+<p>How many circles are there?</p>
+<svg xmlns="http://www.w3.org/2000/svg" role="img" width="100" height="100">
+  <title>1 circle</title>
+	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Passed example 2
+
+The `<circle>` element has an explicit role of `graphics-symbol` and an accessible name from the `aria-label` attribute that does not contain only whitespace.
+
+```html
+<p>How many circles are there?</p>
+<svg xmlns="https://www.w3.org/2000/svg" role="img" aria-label="1 circle">
+	<circle role="graphics-symbol" cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow" aria-label="1 circle">
+</svg>
+```
+
+### Failed
+
+#### Failed example 1
+
+The `<svg>` element has an explicit role of `img` but has no accessible name.
+
+```html
+<p>How many circles are there?</p>
+<svg xmlns="http://www.w3.org/2000/svg" role="img">
+	<circle cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 2
+
+The `<svg>` has an explicit role of `img`but has only whitespace in the `<title>` element for the accessible name.
+
+```html
+<p>How many circles are there?</p>
+<svg xmlns="http://www.w3.org/2000/svg" role="img">
+	<title> </title>
+  	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 3
+
+The `<svg>` element is included in the accessibility tree with an explicit role of `img` but has no content in the `<title>` element for an accessible name.
+ 
+```html
+<p>How many circles are there?</p>
+<svg xmlns="http://www.w3.org/2000/svg" role="img">
+	<title></title>
+  	<circle cx="50" cy="50" r="40" fill="yellow"></circle>
+</svg>
+```
+
+#### Failed example 4
+
+The `<circle>` elements have an explicit role of `graphics-symbol` but do not have accessible names.
+
+```html
+<p>How many circles are there?</p>
+<svg xmlns="http://www.w3.org/2000/svg">
+	<circle role="graphics-symbol" cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
+</svg>
+```
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+The `svg` element does not render any visible content to the screen and is not included in the accessibility tree. 
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100">
+</svg>
+```
+
+#### Inapplicable example 2
+
+The `<svg>` or `<circle>` elements do not have an explicit role.
```

Comment:
or > and

Reply:
changed to...
> Both the `<svg>` and `<circle>` elements do not have an explicit role.

Reply:
Also, the convention in ACT-R is to not use angle bracket around element names. `svg`, `circle`.